mv/core/src/main/scala/org/apache/carbondata/mv/extension/MVHelper.scala

```diff
@@ -568,17 +573,18 @@ object MVHelper {
     generatePartitionerField(allPartitionColumn.toList, Seq.empty)
   }
 
-  private def inheritTablePropertiesFromMainTable(parentTable: CarbonTable,
+  private def inheritTablePropertiesFromMainTable(
```

Comment:
this method we can skip for non carbon tables right?

Reply:
yes, fixed